src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1535,15 +1520,56 @@ Copyright (C) Microsoft Corporation. All rights reserved.
           there is also a .NETFramework implementation.
 
        2. As an escape hatch for cases where the compatibility check performed by 
-          GetTargetFrameworkProperties is faulty.
+          GetReferenceProperties is faulty.
     -->
     <ItemGroup>
       <_MSBuildProjectReferenceExistent Condition="'%(_MSBuildProjectReferenceExistent.SetTargetFramework)' != ''">
-        <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
+        <SkipGetReferenceProperties>true</SkipGetReferenceProperties>
```

Comment:
I see that they are still called TargetFrameworkProperties in places. And still go to %(SetTargetFramework). I think we should rename all or none, and lean towards none for compat.